cli/mk_data/src/generate.rs

```diff
@@ -420,7 +420,7 @@ pub fn run_post_cmd(
     ignore_errors: bool,
 ) -> Result<(), Error> {
     let mut cmd = Command::new("bash");
-    cmd.arg("-c");
+    cmd.arg("-euc");
```

Comment:
> Catch errors from commands other than the very last command

I originally read that as catch errors for commands except for the last, and don't catch errors for the last. But you mean catch errors for commands except for the last, and continue catching errors for the last as we already do, correct?

Reply:
I'll reword it to "in addition to the last".

Reply:
This also made me realise that the change wasn't correct; `-eu` should only be added when `ignore_errors` is false.